idb/postgres/postgres.go

```diff
@@ -130,6 +131,28 @@ func (db *IndexerDb) init() (err error) {
 	return
 }
 
+// Reset is part of idb.IndexerDB
+func (db *IndexerDb) Reset() (err error) {
+	// new database, run setup
+	_, err = db.db.Exec(reset_sql)
+	if err != nil {
+		return fmt.Errorf("db reset failed, %v", err)
+	}
+	db.log.Debugf("reset.sql done")
+	for i, cmd := range registerAlterTableSQLLines {
```

Comment:
Do we really need to alter the tables? If we are removing all accounting data anyway, can we just drop the tables and create new ones with the latest schema versions?

Reply:
Good idea. I guess this would mean changing reset.sql to drop the tables instead of truncating. Do we need anything else?

Reply:
This is probably a good idea. I should branch my branch to write that code and see if it works out well.